### PR TITLE
Update fixtures and add authentication bypass to cypress

### PR DESCRIPTION
**Pull request by norbye**
Fixtures are freshly generated and match the current structure of the database.

Using fixtures of the sessions, the external oauth2 (lego) login can be bypassed for testing purposes, allowing cypress to check more parts of the project.

I had some problems with the CSRF-protection, as the way I'm bypassing the login is basically committing Cross-Site Request Forgery.. might disable CSRF for the test environment to enable submitting forms, but not yet sure if there are any better approaches..

### CODE REVIEW

File: cypress/e2e/submit_application_spec.cy.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  it("successfully apply to one group", () => {
+    cy.login("webkom");
+    cy.visit("/");
+    cy.contains("Gå til søknad").click();
+    cy.contains("Velg komité").click();
+    cy.contains("Gå videre").click();
+    cy.get("input[name='phoneNumber']").type("12345678");
+    cy.contains("Søknadstekst").type("Hei jeg vil gjerne søke");
+    // cy.contains("Send inn søknad").click();
+    // cy.contains("Vi har mottatt søknaden din!").should("exist");
```
Comment (eikhr) [inline cypress/e2e/submit_application_spec.cy.ts:11]: Why is this commented?
Edit: nevermind, it's the CSRF thing right?

**Review by eikhr — COMMENTED** [review]
Hmm, yeah the CSRF thing is kind of a big problem for this.
Correct me if I'm wrong here, but it seems like django uses a CSRF-cookie that changes each time you log in, and then calculates hidden form fields based on the cookie and a random mask. Could we not just save the CSRF-cookie corresponding to this session, and load it together with the sessionId?

File: cypress/e2e/submit_application_spec.cy.ts (hunk 1 above)
Comment (norbye) [reply cypress/e2e/submit_application_spec.cy.ts:11]: Yup, that's the step that's being blocked by CSRF

**Comment by norbye** [comment]
> Correct me if I'm wrong here, but it seems like django uses a CSRF-cookie that changes each time you log in, and then calculates hidden form fields based on the cookie and a random mask. Could we not just save the CSRF-cookie corresponding to this session, and load it together with the sessionId?

That is a very good point! Got a little stuck on not knowing how it is generated, but that sounds smarter. Will play around with it and see if I can get that to work:)

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -32,6 +32,27 @@
...
-// });
+type UserData = {
+  sessionid: string;
+  csrftoken: string;
+};
+
+const sessions: Record<string, UserData> = {
+  webkom: {
+    sessionid: "ln6y4kwaqp3y5mxv1ft19p6or1a018xq",
+    csrftoken: "vKaIGVmBwUPuNkZyvFVy0Ep2IH4B0mG4", // bNnzylsDzcNAY4YZdJL33vai22T7NX0AVqbbHXtJVe7MNt4FeJnV5ZQpu5YbinZU
```
Comment (eikhr) [inline cypress/support/commands.ts:43]: What's up with the commented token?

**Review by eikhr — APPROVED** [review]
Nice!

File: cypress/support/commands.ts (hunk 2 above)
Comment (norbye) [reply cypress/support/commands.ts:43]: Good catch, that was just something I saved while I was playing around - will remove

**Pull request merged**